packages/vscode-extension/tests/specs/webviews/params.spec.ts

```diff
@@ -68,10 +69,11 @@ suite('Params panel testing', () => {
   }
 
   async function forceDisconnect() {
-    await browser.executeWorkbench(async (vscode) => {
+    void browser.executeWorkbench((vscode) => {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
-      await vscode.commands.executeCommand('neo4j.internal.forceDisconnect');
+      void vscode.commands.executeCommand('neo4j.internal.forceDisconnect');
     });
```

Comment:
At some point I was using the `void`s as well, but that meant we had a notification that unless closed, the method wasn't finishing? 

I think we should just swap all of the:

```
await window.showErrorMessage
```

for:

```
void window.showErrorMessage
```

?

Reply:
>we had a notification that unless closed, the method wasn't finishing?
what does this mean? This method still blocks like before due to the new `await waitUntilNotification` below

Reply:
I'm not implying we should change anything here but that we should replace the `await window.showErrorMessage` with `void window.showErrorMessage` everywhere in the codebase? I don't see any reason why that should block the execution in general of some method when things went wrong

Reply:
i mean the test will fail if the show error message goes wrong no? The very next assertion is waiting for it to fire?